pytensor/tensor/basic.py

```diff
@@ -760,7 +760,16 @@ def switch(cond, ift, iff):
     """if cond then ift else iff"""
 
 
-where = switch
+def where(cond, ift=None, iff=None):
+    # Normal switch incase both arguements are passed
+    if ift is not None and iff is not None:
+        return switch(cond, ift, iff)
+    # Add case when only condition is passed
+    elif ift is None and iff is None:
+        return as_tensor(cond).nonzero()
+    # Raise an error if only one arguement is passed
+    else:
+        raise Exception("Either both or none of the parameters should be passed")
```

Comment:
The comments don't seem to add more information then what can be read from the code?

I would remove them.

Instead can you add some docstrings? Mentioning it works like numpy where/copying and adapting their docstrings to mention TensorVariable instead of arrays should be fine.